build.gradle

```diff
@@ -25,8 +25,8 @@ plugins {
 ext.ballerinaLangVersion = project.ballerinaLangVersion
 ext.kafkaVersion = project.kafkaVersion
 ext.awaitalityVersion = "3.1.6"
-ext.avroVersion = "1.9.2"
 ext.puppycrawlCheckstyleVersion = "8.18"
+ext.avroVersion = project.avroVersion
 ext.stdlibFileVersion = project.stdlibFileVersion
 ext.stdlibIoVersion = project.stdlibIoVersion
 ext.stdlibLogVersion = project.stdlibLogVersion
```

Comment:
We can remove all these `ext.*` versions.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/280/commits/a40cc516702bc802ad582fe4dc4d6c5f34ce06f4